tools/crates/vendored_rust_dep.gni

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/tools/crates/cargo.gni")
+
+template("vendored_rust_dep") {
```

Comment:
this name seems a bit odd to me, it sounds like a target that vendors a rust dep. `build_crate`?

Reply:
Sure.